src/components/content-box/_content-box.scss

```diff
@@ -34,7 +34,7 @@ $include-html: false !default;
 
     &--spaced {
       .mint-content-box__content, .mint-content-box__footer {
-        @media (min-width: $breakpointMedium) {
+        @include mintBreakpoint(medium-up) {
```

Comment:
But we do not have a convention for that. 
I do not see a good reason to make this thing and exception from naming convention.